sklearn/tree/tests/test_export.py

```diff
@@ -399,20 +399,39 @@ def test_export_text():
     assert export_text(reg, decimals=1, show_weights=True) == expected_report
 
 
-def test_plot_tree(pyplot):
+def test_plot_tree_entropy(pyplot):
     # mostly smoke tests
-    # Check correctness of export_graphviz
+    # Check correctness of export_graphviz for criterion = entropy
     clf = DecisionTreeClassifier(max_depth=3,
                                  min_samples_split=2,
-                                 criterion="gini",
+                                 criterion="entropy",
                                  random_state=2)
     clf.fit(X, y)
 
     # Test export code
     feature_names = ['first feat', 'sepal_width']
     nodes = plot_tree(clf, feature_names=feature_names)
     assert len(nodes) == 3
-    assert nodes[0].get_text() == ("first feat <= 0.0\nentropy = 0.5\n"
+    assert nodes[0].get_text() == ("first feat <= 0.0\nentropy = 1.0\n"
```

Comment:
Could you remove entropy and instead use criterion. Else LGTM.

Reply:
Will make that change. Thanks!

Reply:
@abhinavsagar this test is for the case where the tree's criterion is indeed entropy. See test_plot_tree_gini below.